src/managers/EmbraceLogManager/EmbraceLogManager.ts

```diff
@@ -58,32 +58,48 @@ export class EmbraceLogManager implements LogManager {
   }
 
   public logException(
-    error: Error,
+    error: unknown,
     {
       handled = true,
       attributes = {},
       timestamp = this._perf.getNowMillis(),
     }: LogExceptionOptions = {}
   ) {
+    if (!error) {
+      error = new Error('undefined logException');
```

Comment:
won't be much we can show in any case but maybe a slightly verbose explanation here like:
```suggestion
      error = new Error('logException received a falsy error');
```